<ui version="4.0" >
 <class>QueryOptions</class>
 <widget class="QWidget" name="QueryOptions" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>527</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" name="queryLayout" >
     <property name="sizeConstraint" >
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <layout class="QGridLayout" name="mainQueryLayout" >
       <item row="1" column="1" >
        <widget class="KLineEdit" name="querySource" >
         <property name="toolTip" >
          <string>Search expression for source language part.
Press Enter to start the search.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="KLineEdit" name="queryTarget" >
         <property name="toolTip" >
          <string>Search expression for target language part.
Press Enter to start the search.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="querySourceLabel" >
         <property name="text" >
          <string>Source:</string>
         </property>
         <property name="buddy" >
          <cstring>querySource</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="queryTargetLabel" >
         <property name="text" >
          <string>Target:</string>
         </property>
         <property name="buddy" >
          <cstring>queryTarget</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QCheckBox" name="invertTarget" >
         <property name="toolTip" >
          <string>Show results that do not match search expression</string>
         </property>
         <property name="text" >
          <string>Invert</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2" >
        <widget class="QCheckBox" name="invertSource" >
         <property name="toolTip" >
          <string>Show results that do not match search expression</string>
         </property>
         <property name="text" >
          <string>Invert</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="optionsLayout" >
       <item>
        <layout class="QHBoxLayout" name="filemaskLayout" >
         <item>
          <widget class="QLabel" name="label_2" >
           <property name="toolTip" >
            <string/>
           </property>
           <property name="text" >
            <string>File mask:</string>
           </property>
           <property name="buddy" >
            <cstring>filemask</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="filemask" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip" >
            <string>Show only results from files with path matching specified mask</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="queryStyleLayout" >
         <item>
          <widget class="QRadioButton" name="substr" >
           <property name="toolTip" >
            <string>Case insensitive</string>
           </property>
           <property name="text" >
            <string comment="@option:radio" >Substring</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="like" >
           <property name="toolTip" >
            <string>Space is AND operator. Case insensitive.</string>
           </property>
           <property name="text" >
            <string comment="@option:radio" >Google-like</string>
           </property>
           <property name="checked" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="glob" >
           <property name="toolTip" >
            <string>Shell globs (* and ?). Case sensitive.</string>
           </property>
           <property name="text" >
            <string comment="@option:radio" >Wildcard</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="buttonsLayout" >
       <item>
        <widget class="QPushButton" name="doFind" >
         <property name="text" >
          <string>&amp;Search</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="doUpdateTM" >
         <property name="toolTip" >
          <string>Reads all project files into translation memory, updating old entries. The scanning will work in background.</string>
         </property>
         <property name="text" >
          <string>&amp;Rescan Project Files</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="alternatingRowColors" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>querySource</tabstop>
  <tabstop>queryTarget</tabstop>
  <tabstop>doFind</tabstop>
  <tabstop>invertSource</tabstop>
  <tabstop>invertTarget</tabstop>
  <tabstop>filemask</tabstop>
  <tabstop>substr</tabstop>
  <tabstop>like</tabstop>
  <tabstop>glob</tabstop>
  <tabstop>doUpdateTM</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
